python/lsst/daf/butler/instrument.py

```diff
@@ -96,12 +96,11 @@ def makeExposureEntryFromVisitInfo(dataId, visitInfo, snap=0):
     """
     avg = visitInfo.getDate()
     begin = DateTime(int(avg.nsecs(DateTime.TAI) - 0.5E9*visitInfo.getExposureTime()), DateTime.TAI)
+    end = DateTime(int(avg.nsecs(DateTime.TAI) + 0.5E9*visitInfo.getExposureTime()), DateTime.TAI)
```

Comment:
This is an undocumented dependency on `daf_base`. I'm a bit worried about this code relying on afw and daf_base being inside butler.

Is this going to be replaced by obs_metadata?

Reply:
It'll be replaced by obs_metadata and may be moved to obs_base even before that happens.